memcache/memcache.go

```diff
@@ -711,3 +739,238 @@ func (c *Client) incrDecr(verb, key string, delta uint64) (uint64, error) {
 	})
 	return val, err
 }
+
+type MetaSetItem struct {
+	Key   string
+	Flags MetaSetFlags
+	Value []byte
+}
+
+type MetaResponseMetadata struct {
```

Comment:
I have pushed my final changes. Lets reconcile these values so that both of our code has same names. You can find my latest changes here https://github.com/spaiStrp/gomemcache/blob/789be43a1e3958b622c53ef4a8fb83828d7fdb4c/memcache/memcache.go#L198 . Let's do the same for MetaSet flags. Open to change the names if you feel some of these are not correct.